nucypher/blockchain/eth/registry.py

```diff
@@ -101,6 +101,11 @@ def write(self, registry_data: list) -> None:
     def read(self) -> Union[list, dict]:
         raise NotImplementedError
 
+    @classmethod
```

Comment:
This shouldn't be a `classmethod`, are you thinking of a `staticmethod`?

Reply:
This method accesses class attributes, so I 'd say it's :ok_hand: .

Reply:
Yes, I don't get why it should be a `staticmethod` if it uses 2 class attributes. The same applies for the other exisiting `classmethod`s in that class.